src/Literal.svelte

```diff
@@ -11,34 +15,32 @@ if (!i18nState) {
   throw new Error(`Literal requires {i18nState} to be passed as property.`)
 }
 
-function log(nodes) {
-  nodes.forEach(node => {
-    if (node instanceof CustomComponent) {
-      console.log('CustomComponent:', node.Component.prototype.constructor, JSON.stringify(node.props))
-    } else if (node instanceof Html) {
-      console.log('Html:', JSON.stringify(node.props))
-    }
-  })
-}
-
+$: status = $i18nAsyncStatus
 $: resolvers = i18nState.resolvers(namespace)
 $: nodes = $resolvers.nodes(path, modifiers)
-
-// $: log(nodes)
+$: scopedNamespace = i18nState.resolveNamespace(namespace)
+$: resolvedKeyPath = (scopedNamespace ? scopedNamespace + '.' + path : path)
+$: resolvedKey = $resolvers.text(path, modifiers)
+$: isResolved = resolvedKeyPath !== resolvedKey
```

Comment:
Even if the status is 'Loading' in general, if the key is resolved, there is no need to display the loading content. The resolved value is displayed.